core/domain/stats_domain.py

```diff
@@ -93,28 +101,51 @@ def validate(self):
                 'Expected exp_version to be an int, received %s' % (
                     self.exp_version))
 
-        if not isinstance(self.num_starts, int):
+        if not isinstance(self.num_starts_v1, int):
```

Comment:
At this point you might consider pulling out a separate internal method like require_nonnegative_int(self.num_starts_v1). Too much duplication otherwise.

Reply:
Done.